File: code/generate_strategies/RA(name)/java/java/function_asyncio_create_task.java
package com.example;

import reactor.core.publisher.Mono;
import java.time.Duration;

public class AsyncTaskHandler {
    public String run_task(String name, int delay) {
        class AsyncTaskHandler {
            public String run_task(String name, int delay) {
                if (delay < 0) {
                    throw new IllegalArgumentException("Delay must be a non-negative integer");
                }

                try {
                    return Mono.just(name)
                            .delayElement(Duration.ofSeconds(delay))
                            .map(n -> String.format("Task %s completed after %d seconds", n, delay))
                            .block();
                } catch (Exception e) {
                    throw new RuntimeException("Failed to execute async task", e);
                }
            }
        }
        return new AsyncTaskHandler().run_task(name, delay);
    }
}

File: code/generate_strategies/RA(name)/python/python/function_asyncio_create_task.py
import asyncio

class AsyncTaskHandler:
    def run_task(self, name: str, delay: int) -> str:
        
        if delay < 0:
            raise ValueError("Delay must be a non-negative integer")

        async def async_task(name: str, delay: int) -> str:
            await asyncio.sleep(delay)
            return f"Task {name} completed after {delay} seconds"

        loop = asyncio.new_event_loop()
        asyncio.set_event_loop(loop)
        try:
            result = loop.run_until_complete(async_task(name, delay))
        finally:
            loop.close()
        return result
